webcompat/helpers.py

```diff
@@ -456,3 +459,29 @@ def api_request(method, path, params=None, data=None):
                 get_response_headers(resource))
     else:
         abort(404)
+
+
+def cache_policy(private=True, uri_max_age=3600):
```

Comment:
Can we add a comment about what the default `uri_max_age=3600` represents? HTTP noobs like me have to look it up.

(The `private` arg has a nice inline comment)